Source/WebCore/page/SecurityOrigin.cpp

```diff
@@ -396,8 +399,11 @@ bool SecurityOrigin::isSameOriginAs(const SecurityOrigin& other) const
     if (this == &other)
         return true;
 
-    if (isOpaque() || other.isOpaque())
-        return data().opaqueOriginIdentifier == other.data().opaqueOriginIdentifier;
+    if (m_data.data.index() != other.m_data.data.index())
```

Comment:
This is another example where the new code is much more complicated and shouldn't need to be.